<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectToStadiaWidget</class>
 <widget class="QWidget" name="ConnectToStadiaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Orbit Profiler</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="accessibleName">
         <string>ConnectToStadia</string>
        </property>
        <property name="text">
         <string>Connect to a Stadia instance</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contentFrame">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>198</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="orbit_session_setup::RetrieveInstancesWidget" name="retrieveInstancesWidget" native="true">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTableView" name="instancesTableView">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="accessibleName">
                   <string>InstanceList</string>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <property name="showGrid">
                   <bool>false</bool>
                  </property>
                  <property name="wordWrap">
                   <bool>false</bool>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                 </widget>
                </item>
               </layout>
               <widget class="orbit_session_setup::OverlayWidget" name="instancesTableOverlay" native="true">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>100</width>
                  <height>30</height>
                 </rect>
                </property>
                <property name="visible">
                 <bool>false</bool>
                </property>
                <property name="accessibleName">
                 <string>InstanceListOverlay</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
               <item>
                <widget class="QCheckBox" name="rememberCheckBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="toolTip">
                  <string>Connect to the selected instance on Orbits next start</string>
                 </property>
                 <property name="accessibleName">
                  <string>AutoConnect</string>
                 </property>
                 <property name="text">
                  <string>Automatically connect to this instance next time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="connectButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="accessibleName">
                  <string>ConnectButton</string>
                 </property>
                 <property name="text">
                  <string>Connect</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>orbit_session_setup::OverlayWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/OverlayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>orbit_session_setup::RetrieveInstancesWidget</class>
   <extends>QWidget</extends>
   <header>SessionSetup/RetrieveInstancesWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
